portfolio/src/main/webapp/script.js

```diff
@@ -114,96 +114,36 @@ function viewReplies() {
   });
 }
 
-const LOCATIONS = [
-  {
-    name: 'The Twelve Apostles',
-    lat: -38.6621,
-    lng: 143.1051
-  },
-  {
-    name: 'Sydney Opera House',
-    lat: -33.8568,
-    lng: 151.2153
-  },
-  {
-    name: 'Rocky Mountains National Park',
-    lat: 40.3428,
-    lng: -105.6836
-  },
-  {
-    name: 'Sydney Opera House',
-    lat: -33.8568,
-    lng: 151.2153
-  },
-  {
-    name: 'Channel Islands National Park', 
-    lat: 34.0069,
-    lng: -119.7785
-  },
-  {
-    name: 'Zion National Park', 
-    lat: 37.2982,
-    lng: -113.0263
-  },
-  {
-    name: 'Bryce Canyon National Park', 
-    lat: 37.5930, 
-    lng: -112.1871
-  },
-  {
-    name: 'Arches National Park', 
-    lat: 38.7331, 
-    lng: -109.5925
-  },
-  {
-    name: 'Canyonlands National Park', 
-    lat: 38.3269, 
-    lng: -109.8783
-  },
-  {
-    name: 'Death Valley National Park', 
-    lat: 36.5054, 
-    lng: -117.0794
-  },
-  {
-    name: 'Capitol Reef National Park', 
-    lat: 38.3670, 
-    lng: -111.2615
-  },
-  {
-    name: 'Pinnacles National Park', 
-    lat: 36.4906, 
-    lng: -121.1825
-  },
-  {
-    name: 'Joshua Tree National Park', 
-    lat: 33.8734, 
-    lng: -115.9010
-  },
-  {
-    name: 'Petrified Forest National Park', 
-    lat: 34.9100, 
-    lng: -109.8068
-  },
-  {
-    name: 'Scripps College', 
-    lat: 34.1038, 
-    lng: -117.7110
-  },
-  {
-    name: 'Antelope Valley, California', 
-    lat: 34.7514, 
-    lng: -118.2523
-  }];
+function makeLocation(locName, latitude, longitude) {
```

Comment:
Imo I would make this an actual class.
I guess it doesn't matter much here, but with most typing systems you will get much better type safety from using classes than just plain objects.  We try to avoid plain objects for the most part unless we truly need a generic object that could have a variety of different params

Reply:
Good to know, will definitely keep this in mind going forward!